lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -1006,88 +1013,6 @@ export abstract class CacheManager implements ICacheManager {
         await Promise.all(removedCredentials);
     }
 
-    /**
-     * Migrates a single-tenant account and all it's associated alternate cross-tenant account objects in the
-     * cache into a condensed multi-tenant account object with tenant profiles.
```

Comment:
Any changes a user would notice in MTA support?

Reply:
No, old cache will be cleared anyway on init.